### DISCUSSION TITLE
How can i subscribe a list record?

### DISCUSSION THREAD

**Discussion by yamadaOliva** [pocketbase/pocketbase#5218] [Q&A] [answered]
I want to know the change in the product list with code "abc". What should i do, subscribe each record ? how can i use query for that prolem ?

**Comment by ganigeorgiev** [comment, marked as answer]
You can pass `filter` as option to the subscribe call. For example:

```js
pb.collection("example").subscribe("*", (e) => {
   ...
}, { filter: "code='abc'" })
```

**Reply by yamadaOliva** [reply]
i used API realtime not like database sir
